WordPress/gutenberg
# FocalPointPicker: Fix layout misalignment when placed in the `BlockInspector`

## What?
This PR fixes a problem with the unit controls (Left, Top) not being aligned vertically when the `FocalPointPicker` component is used in the block sidebar (`InspectorControls`).

| Before | After |
| --- | --- |
| ![before](https://user-images.githubusercontent.com/54422211/208288898-6919e892-4ab3-422b-8910-267a29a1af72.png) | ![after](https://user-images.githubusercontent.com/54422211/208288902-cc36af4e-c038-4055-91e2-9940aa28a63e.png) |

## Why?

The control itself has no margin, but `margin-bottom` is applied by the `InspectorControl`. And this is because the value is smaller in the UnitControl on the right side.

![focalpointpicker](https://user-images.githubusercontent.com/54422211/208288647-a0a42658-6d5a-4158-b529-f30ddee03aea.png)

## How?

I have reset the `margin-bottom` on the `UnitControl`. I believe this is the appropriate approach as this margin is not expected, but if there is another way, please advise.

## Testing Instructions

- Open the post/site editor.
- Insert a cover or media & text block, which has this component in the sidebar.
- Confirm that the vertical positions of the left and top controls are aligned.
